packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -510,6 +510,9 @@ const disableVmHighAvailability = async (xapi: Xapi, vm: Vm) => {
 // │  │  ├─ task.start(message: 'merge')
 // │  │  │  ├─ task.warning(message: string)
 // │  │  │  └─ task.end(result: { size: number })
+// │  │  ├─ task.start(message: 'clean')
```

Comment:
It should be documented in which case it happens.

And probably also for the `merge` task.